ui.js

```diff
@@ -136,7 +278,14 @@ function Equation(eqn){
 		$(this.tile).remove()
 		this.e=this.tile=this.visibilitybtn=this.input=this.removebtn=null
 		equations.remove(this)
-		if (equations.length<8) $('#add').show()
+		if (spaceAvailable() === 1){
+			$('#add-implicit').show();
+		} else if(spaceAvailable() === 2){
```

Comment:
I would just keep it consistent and show/hide all buttons at once, so people don't wonder why the vector/parametric functionality suddenly disappeared